src/avrt.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright © 2016, Baptiste AUBRY
+// Licensed under the MIT License <LICENSE.md>
+
+ENUM_NEG!{enum AVRT_PRIORITY {
+    AVRT_PRIORITY_VERYLOW = -2,
+    AVRT_PRIORITY_LOW,
+    AVRT_PRIORITY_NORMAL,
+    AVRT_PRIORITY_HIGH,
+    AVRT_PRIORITY_CRITICAL,
+}}
+
+ENUM!{enum AVRT_NOTIFICATION {
```

Comment:
My bad. My visual studio was not properly setup to use the latest sdk... Hence I removed AVRT_NOTIFICATION
The rest doesn't seem to have been modified